app/assets/stylesheets/search.scss

```diff
@@ -137,6 +137,13 @@
   font-weight: 500;
 }
 
+.username-contain {
```

Comment:
Prefix classname with search. Maybe add text-overflow: ellipsis as well to signify shortened display name

Reply:
made changes accordingly, resolved!!